packages/p2p/src/components/orders/orders.jsx

```diff
@@ -109,6 +110,7 @@ const Orders = ({ params, navigate, chat_info }) => {
 
 Orders.propTypes = {
     chat_info: PropTypes.object,
+    order_details: PropTypes.object,
```

Comment:
`order_details` is state rather than a prop.

Reply:
You're right! My bad.